superset/commands/dashboard/delete.py

```diff
@@ -37,6 +38,19 @@
 logger = logging.getLogger(__name__)
 
 
+class DeleteEmbeddedDashboardCommand(BaseCommand):
+    def __init__(self, dashboard: Dashboard):
+        self._dashboard = dashboard
+
+    @transaction(on_error=partial(on_error, reraise=DashboardDeleteEmbeddedFailedError))
+    def run(self) -> None:
+        self.validate()
+        return EmbeddedDashboardDAO.delete(self._dashboard.embedded)
+
+    def validate(self) -> None:
+        pass
```

Comment:
Should we include some minimal validations like: the model exist, etc?

Reply:
I am not sure what are the implications here. This had no checks originally and I could not find documentation about the expected behavior. So I would not want to touch it unless we have a clear definition of the expected behavior.